tensornetwork/contractors/opt_einsum_paths/path_contractors_node_test.py

```diff
@@ -142,3 +142,33 @@ def test_single_node_reorder(backend, path_algorithm):
   result = path_algorithm({a}, expected_edge_order)
   assert result.edges == expected_edge_order
   np.testing.assert_allclose(result.tensor, np.arange(4).reshape((2, 2)).T)
+
+
+def test_ignore_edge_order(backend, path_algorithm):
+  a = tn.Node(np.ones((1, 1, 1)), backend=backend)
+  b = tn.Node(np.ones((1, 1, 1, 2, 3)), backend=backend)
+
+  a[0] ^ b[0]
+  a[1] ^ b[1]
+  a[2] ^ b[2]
+
+  final_node = path_algorithm({a, b},
+                              ignore_edge_order=True)
+  assert final_node.tensor.shape == (2, 3)
```

Comment:
It's unlikely that the final shape will remain consistent with `ignore_edge_order=True`. Can you instead test that `set(final_nodes.edges) == {e[0], e[1]}` (same for below.)